extensions/skins/conversation_v1/player.html

```diff
@@ -2,107 +2,258 @@
+            <answer-feedback-pair profile-picture="profilePicture"
+                                  answer="activeCard.answerFeedbackPairs[activeCard.answerFeedbackPairs.length - 1].learnerAnswer"
+                                  short-answer="activeCard.answerFeedbackPairs[activeCard.answerFeedbackPairs.length - 1].shortLearnerAnswer"
+                                  feedback="activeCard.answerFeedbackPairs[activeCard.answerFeedbackPairs.length - 1].oppiaFeedback">
+            </answer-feedback-pair>
+          </div>
+
+          <div ng-if="!waitingForOppiaFeedback && (activeCard.interaction || waitingForContinueButtonClick) && currentProgressDotIndex === transcript.length - 1">
+            <div class="conversation-skin-inline-interaction">
+              <img class="conversation-skin-user-avatar img-circle" ng-src="<[profilePicture]>">
+              <div ng-if="waitingForContinueButtonClick">
+                <md-button class="md-button-success protractor-test-continue-to-next-card-button" focus-on="<[CONTINUE_BUTTON_FOCUS_LABEL]>"
+                           ng-click="showPendingCard(upcomingStateName, upcomingContentHtml)">
+                  Continue
+                </md-button>
+              </div>
+              <div ng-if="activeCard.interaction && !activeCard.interactionIsDisabled">
+                <div ng-if="activeCard.interactionIsInline">
+                  <div class="protractor-test-conversation-input"
+                       angular-html-bind="activeCard.interaction">
+                  </div>
+                </div>
+                <div ng-if="!activeCard.interactionIsInline" ng-click="setVisiblePanel(PANEL_INTERACTION)" ng-class="{'conversation-skin-interaction-clickable-instructions': panels.length > 1}" style="opacity: 0.8;">
+                  (<[activeCard.interactionInstructions]>)
+                  <i class="zmdi zmdi-arrow-right"></i>
+                </div>
               </div>
             </div>
-          </md-card>
+          </div>
         </div>
-      </div>
+      </md-card>
 
-      <div ng-if="waitingForNewCard">
-        <div class="conversation-skin-loading-dots">
-          <div class="oppia-loading-dot-one"></div>
-          <div class="oppia-loading-dot-two"></div>
-          <div class="oppia-loading-dot-three"></div>
-        </div>
+      <div class="conversation-skin-supplemental-card-container" ng-class="{'conversation-skin-right-card-container': !activeCard.interactionIsInline && isPanelVisible(PANEL_TUTOR) && isPanelVisible(PANEL_INTERACTION)}" ng-if="!activeCard.interactionIsInline && activeCard.interaction">
+        <md-card ng-show="isPanelVisible(PANEL_INTERACTION)"
+                 class="conversation-skin-supplemental-card">
+          <div ng-if="activeCard.interactionIsDisabled" class="conversation-skin-supplemental-card-overlay">
+          </div>
+          <div ng-repeat="transcriptCard in transcript">
```

Comment:
Does this mean multiple interactions will be visible side-by-side if they fit onto the screen?